modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-portlet/build.gradle

```diff
@@ -1,10 +1,9 @@
 dependencies {
-	compileInclude group: "com.liferay", name: "org.jsoup", version: "1.10.2.LIFERAY-PATCHED-3"
-
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.impl", version: "default"
 	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
 	compileOnly group: "javax.portlet", name: "portlet-api", version: "3.0.1"
 	compileOnly group: "org.apache.felix", name: "org.apache.felix.http.servlet-api", version: "1.1.2"
+	compileOnly group: "org.jsoup", name: "jsoup", version: "1.15.1"
```

Comment:
@jeffwu0724 With this change, you are not going to include the patched version. Are you sure you do not need to include the patch after upgrade?

Reply:
Please search "1.10.2.LIFERAY-PATCHED-3", you will find it is added for https://issues.liferay.com/browse/LPS-81724 to fix a bug, please make sure this bug is fixed after you upgrade to 1.15 without the patched code